notebooks/official/migration/sdk-custom-image-classification-prebuilt-container.ipynb

```diff
@@ -1575,7 +1458,7 @@
         "id": "endpoints_predict:migration,new,mbsdk"
       },
       "source": [
-        "### [predictions.online-prediction-automl](https://cloud.google.com/vertex-ai/docs/predictions/online-predictions-automl)"
+        "Learn more about how to [Train and use your own models](https://cloud.google.com/vertex-ai/docs/predictions/online-predictions-automl)"
```

Comment:
The URL involves a redirect.
- update from: https://cloud.google.com/vertex-ai/docs/predictions/online-predictions-automl
- to: https://cloud.google.com/vertex-ai/docs/training-overview